WBooks/BookDetails/BookDetailsController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  BookDetailsController.swift
+//  WBooks
+//
+//  Created by Juan Silva on 14/06/2021.
+//
+
+import UIKit
+
+final class BookDetailsController: UIViewController {
+    private lazy var bookDetailsView = BookDetailsView()
+    private let infoBookSectionController: InfoBookSectionController
+
+    
+    init(viewModel: BookDetailsViewModel) {
+        let infoBookSectionViewModel = viewModel.createInfoBookSectionViewModel()
+        infoBookSectionController = InfoBookSectionController(bookSectionViewModel: infoBookSectionViewModel)
```

Comment:
This is just a refactor, but if you'd like to extract the creation of `InfoBookSectionController` from the initializer, you could move that code to a lazy property with a closure:

```Swift
...
private let viewModel: BookDetailsViewModel
private lazy var infoBookSectionController: InfoBookSectionController = {
    let sectionViewModel = viewModel.createInfoBookSectionViewModel()
    return InfoBookSectionController(bookSectionViewModel: sectionViewModel)
}()

init(viewModel: BookDetailsViewModel) {
    self.viewModel = viewModel
    super.init(nibName: nil, bundle: nil)
}
```

This is not better or worse, just another way of organizing the code. The good thing is that if you have many "sections" (and thus many controllers), the code will be separated neatly. 😎